simplegrid/cow.py

```diff
@@ -37,20 +38,21 @@ def to_observation(self, offset):
         return direction[0] + offset, direction[1] + offset
 
 
-class AbstractCow(abc.ABC):
+class AbstractCreature(abc.ABC):
     id_count = 1
 
     def __init_subclass__(cls, **kwargs):
         if not hasattr(cls, 'COLOR'):
-            raise TypeError('Cows need to declare their color')
+            raise TypeError('Creatures need to declare their color')
 
     def __init__(self, x, y, settings, energy=None):
         self.x = x
         self.y = y
         self.settings = settings
         self.energy = energy or settings.init_energy
-        self.id = AbstractCow.id_count
-        AbstractCow.id_count += 1
+        self.id = AbstractCreature.id_count
+        self.is_predator = 0
```

Comment:
What is this? Why is it an int?

Reply:
Line should be deleted. IS_PREDATOR is now a property of the class.